src/components/TimerPlayer/TimerPlayer.js

```diff
@@ -63,20 +66,28 @@ const TimerPlayer = ({ id, timer, deleteTimer }) => {
       interval = setInterval(() => {
         setRemainingTime(remainingTime => remainingTime - 1);
```

Comment:
Probably not something for this PR but its probably worth using actual date values in your calculations rather than having numbers and removing 1. This is probably fine for now but over long periods of time it will probably go out of sync.